components/gateway/internal/auditlog/service.go

```diff
@@ -146,8 +180,6 @@ func (svc *Service) parseResponse(response string) (model.GraphqlResponse, error
 func (svc *Service) createConfigChangeMsg(claims proxy.Claims, request string) model.ConfigurationChange {
 	msg := svc.msgFactory.CreateConfigurationChange()
 	msg.Object = model.Object{ID: fillID(claims, "Config Change")}
-	msg.Attributes = []model.Attribute{
```

Comment:
Better leave this as it was. Also remove the places where this attribute is added by hand